docs: Update README.md (#18)
subhoghoshX/laureate: README.md

# Laureate

A web app that helps you to transform your tweets into beautiful images with different gradient and color combinations.

[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)

## Image of the application (Demo)

![Laureate](https://user-images.githubusercontent.com/107422421/193412168-dc91299b-678a-4781-9351-2cfc1108b5a1.png)

## Table of Contents

* [Demo](#demo)
* [Tech Stack](#tech-stack)
* [Installation](#installation)
* [Contributing](#contributing)
* [License](#license)

## Demo

Provide links to the deployed application for users to be able to view it easily. If you have any other links you would like to show here, feel free.

> View the application at [Deployed Project Link](Link)

Reviewer: ```suggestion
View the application at [Deployed Project Link](https://laureate.netlify.app/)
```

## Tech Stack

This is where you want to demonstrate the technologies you used in this project and provide any links.
| Technology | Description                               | Link ↘️          |
|------------|-------------------------------------------|--------           |
| NextJs     | The React Framework                       |https://nextjs.org/|
|Tailwind CSS| Utility-first CSS framework               |https://tailwindcss.com/|
| Zustand    | Bear necessities for state management     |https://github.com/pmndrs/zustand|
| Twitter api| For fetching tweet data                   |https://developer.twitter.com/en/docs/twitter-api| 

## Installation

> 1. [Fork](https://github.com/Dun-sin/laureate/fork) the project. Click on the fork icon in the top right to get started

Reviewer: ```suggestion
1. [Fork](https://github.com/subhoghoshX/laureate/fork) the project. Click on the fork icon in the top right to get started
```

2. Clone the project, you can use the following command:

   ```bash
   git clone https://github.com/<your-github-username>/laureate
   ```

3. Navigate to the project directory

   ```bash
   cd laureate
   ```

> 4. Install dependencies with `npm install` in both client and server folder

Reviewer: ```suggestion
4. Install dependencies with `yarn install` in both client and server folder
```

   ```bash

>    npm install

Reviewer: ```suggestion
   yarn install
```

   ```

> 5. Run `npm start`

Reviewer: ```suggestion
5. Run `yarn start`
```

   ```bash

>    npm start

Reviewer: ```suggestion
   yarn start
```

   ```
 
## Contributing

- Contributions make the open source community such an amazing place to learn, inspire, and create.
- Any contributions you make are greatly appreciated.
- Check out our [contribution guidelines](/CONTRIBUTING.md) for more information.

> 

Reviewer: ```suggestion

## Pull requests

**Please ask first before starting work on any significant new features.**

It's never a fun experience to have your pull request declined after investing a lot of time and effort into a new feature. To avoid this from happening, we request that contributors create [a feature request](https://github.com/subhoghoshX/laureate/discussions/new) to first discuss any significant new ideas.
```
